cmd/tikv-ctl/src/main.rs

```diff
@@ -1050,867 +1051,661 @@ fn warning_prompt(message: &str) -> bool {
     }
 }
 
-fn main() {
-    let raw_key_hint: &'static str = "Raw key (generally starts with \"z\") in escaped form";
+static VERSION_INFO: SyncLazy<String> = SyncLazy::new(|| {
```

Comment:
Is it required to use `SyncLazy`? Can we do something like `version = get_version_info()`?

Reply:
Yes it’s required. `App::version` takes a `&str`, but `get_version_info()` returns a `String` because of the use of `format!` and `str::trim`. It must have a static variable to hold ownership of the version string.